### ISSUE TITLE
Find a JSON library? Stop using INI

### ISSUE THREAD

**Issue by keli5** [keli5/kcsh#25]

**Comment by keli5** [comment]
1. https://github.com/nlohmann/json

**Comment by sadanslargehole** [comment]
> 1. https://github.com/nlohmann/json

this gives me a build error only on aarch64
```
aarch64-linux-gnu-g++ -o ./build/debug/kcsh-aarch64-debug -std=c++20 -Wall -Wextra -g3 -DNDEBUG main.cpp
In file included from main.cpp:3:
shellutil/settingsutil.hpp:12:10: fatal error: nlohmann/json.hpp: No such file or directory
   12 | #include <nlohmann/json.hpp>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:35: aarch64-debug] Error 1
```
